tests/system/python/packages/test_eds.py

```diff
@@ -0,0 +1,186 @@
+# -*- coding: utf-8 -*-
+
+# FLEDGE_BEGIN
+# See: http://fledge.readthedocs.io/
+# FLEDGE_END
+
+""" Test EDS
+"""
```

Comment:
Please add docstring for pre-requisiste setup, online reference of EDS; and how to run it, If and setup a  CI job for the same using fledge nightly package on Ubuntu 18.04.

Reply:
online reference i.e. https://osisoft.github.io/Edge-Data-Store-Docs/V1/index.html